### DISCUSSION TITLE
I have a question. I recently studied the API. When getting content through the interface, the picture prompts: Logged in session not found.

### DISCUSSION THREAD

**Discussion by openapphub** [TriliumNext/Trilium#4825] [Q&A]
Client access is working fine:：
<img width="747" alt="image" src="https://github.com/user-attachments/assets/2be4f4fb-a1d8-4c16-afe0-904b562ad9be">

Obtain the img label node through the `/notes/{noteId}/content` interface`<img style="aspect-ratio:1012/1185;" src="api/attachments/OfFAHS3RFFOC/image/8-8.jpg" alt ="image.png" width="1012" height="1185">`. Through the reverse proxy, point the image address to the deployment address, and add the request header `Authorization`, but the prompt Logged in session not found is still displayed. Only when developing locally, point the proxy to local http://127.0.0.1:37840 for access.

Do you know how to deal with this?

**Comment by openapphub** [comment]
@rauenzi

**Reply by zerebos** [reply]
Why did you ping/mention me?

**Reply by openapphub** [reply]
> Why did you ping/mention me?

Sorry for the direct reference to you, I follow you and know that you write two projects about api, and would like to ask if you have ever encountered this problem. I was in the process of organizing my language and you replied. If you mind, I can re-edit it. Finally, I apologize.
